templates/vue/src/components/SettingsModal.vue

```diff
@@ -112,10 +120,14 @@ export default {
       userId: "",
       showAccess: true,
       defaultPermissions,
+      fileUploading: false,
     }
   },
   computed: {
     ...mapGetters(["settings", "tapestryJson"]),
+    accessSave() {
+      return !this.fileUploading
+    },
```

Comment:
I don't think this extra computed property is necessary, you can just reference `fileUploading` directly in the template.

Reply:
I might have to read vue docs for more understanding on this, although I agree its redundant